api-reference/beta/api/domain_verify.md

```diff
@@ -0,0 +1,73 @@
+# domain: verify
+
+Validates the ownership of the domain.
+
+> **Important:** Only applies to an unverified domain. For an unverified domain, the isVerified property of the [domain](../resources/domain.md) is false.
+
+### Prerequisites
+
+The following **scopes** are required to execute this API: *Directory.Read.All* or *Domain.ReadWrite.All*
+
+### HTTP request
+
+<!-- { "blockType": "ignored" } -->
+```http
+POST /domains/{id}/verify
+```
+
+> For {id}, specify the domain with its fully qualified domain name.
+
+### Request headers
+
+| Name       | Description|
+|:---------------|:----------|
+| Authorization  | Bearer &lt;code&gt; Required|
```

Comment:
This should be "token" not "code".  Not sure if you have this elsewhere in this PR